### PR TITLE
Add example project

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

File: examples/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    val utilitiesProjectName = "org.jetbrains.research.pluginUtilities"
+    dependencies {
+        implementation(kotlin("stdlib-jdk8"))
+
+        // Plugin utilities modules
+        implementation("$utilitiesProjectName:plugin-utilities-core") {
+            version {
+                branch = "add-python-sdk-util"
```
Comment (Furetur) [inline examples/build.gradle.kts:37]: But what branch should be specified if we want to test our library? `$CURRENT_BRANCH`? 0_0

**Review by Furetur — COMMENTED** [review]
Great! But is it just an example or we are planning on running it on CI?

File: examples/build.gradle.kts (hunk 1 above)
Comment (nbirillo) [reply examples/build.gradle.kts:37]: I am working on this now in another branch, please, wait for a little...

Comment (nbirillo) [reply examples/build.gradle.kts]: @Furetur see it)

File: examples/examples-plugin/src/main/kotlin/org/jetbrains/research/pluginUtilities/examples/JvmProjectOpenerStarter.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                ProjectManagerEx.getInstanceEx().openProject(
+                    projectPath,
+                    OpenProjectTask(isNewProject = true, runConfigurators = true, forceOpenInNewFrame = true)
+                )?.let { project ->
+                    try {
+                        println("The project ${project.basePath} was opened!")
+                    } catch (ex: Exception) {
+                        println(ex)
+                    } finally {
+                        ApplicationManager.getApplication().invokeAndWait {
```
Comment (Furetur) [inline examples/examples-plugin/src/main/kotlin/org/jetbrains/research/pluginUtilities/examples/JvmProjectOpenerStarter.kt:26]: You close a project inside `invokeAndWait`?

**Review by Furetur — COMMENTED** [review]
Everything's cool! I just have 1 question because I don't use `invokeAndWait` in RepositoryOpener

**Pull request merged**